src/lib/Load.js

```diff
@@ -0,0 +1,21 @@
+export default class Load {
```

Comment:
could it make sense to keep load and save part of a single module or a single class?

Reply:
Yeah definitely make sense. :-)